src/calendar/grid/index.tsx

```diff
@@ -53,6 +53,7 @@ export default function Grid({
     }
 
     switch (event.keyCode) {
+      case KeyCode.space:
```

Comment:
Added this for consistency with DRP where the date can be selected with both SPACE and ENTER.

Reply:
If the logic between two components is the same, why is it not a `useKeyboardNavigation` hook or something similar?

Reply:
I like the idea and will check if that can be done as a follow-up 👍 

Hopefully we can reuse more code between calendar and drp calendar eventually, currently the implementations have a lot of inconsistencies which can probably be resolved.